### DISCUSSION TITLE
Alarm on Downtime

### DISCUSSION THREAD

**Discussion by theAkito** [theAkito/webmon#48] [Ideas]
Follow-up on https://github.com/theAkito/webmon/discussions/46#discussioncomment-6897619.

@martin-braun

Could you please elaborate on what behaviour you would expect from that?

Do you want an alarm that rings every day at, for example, 6pm, if a website is down?
Because, if you want instant (14 minutes at worst) feedback, your phone receives a notification, anyway.

**Comment by martin-braun** [comment]
@theAkito Thank you for the notice. Right now the app sends a notification if I'm not mistaken. This can be missed. I don't check my phone every few minutes, so it would be beneficial to get an alarm sound that never stops until an interaction happens.

Like your wake-up alarm or receiving a call by a caller who never gives up on you actually answering the phone. The only way such alarm should be disabled is by configured timeout or interaction (like a wake alarm).

The alarm sound should be configured, so I can pick any annoying sound if I want to.

**Reply by theAkito** [reply]
Ah, I see.

I'll look into that. 🙂
